src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Employee/EmployeeType.php

```diff
@@ -109,63 +137,89 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('firstname', TextType::class, [
+                'label' => $this->trans('First name', 'Admin.Global'),
                 'constraints' => [
                     $this->getNotBlankConstraint(),
                     $this->getLengthConstraint(FirstName::MAX_LENGTH),
                 ],
             ])
             ->add('lastname', TextType::class, [
+                'label' => $this->trans('Last name', 'Admin.Global'),
                 'constraints' => [
                     $this->getNotBlankConstraint(),
                     $this->getLengthConstraint(LastName::MAX_LENGTH),
                 ],
             ])
-            ->add('email', EmailType::class, [
-                'constraints' => [
-                    $this->getNotBlankConstraint(),
-                    $this->getLengthConstraint(EmployeeEmail::MAX_LENGTH),
-                    new Email([
-                        'message' => $this->trans('This field is invalid', [], 'Admin.Notifications.Error'),
-                    ]),
-                ],
-            ])
             ->add('avatarUrl', FileType::class, [
+                'label' => $this->trans('Avatar', 'Admin.Global'),
                 'required' => false,
                 'attr' => [
                     'accept' => 'gif,jpg,jpeg,jpe,png',
```

Comment:
Should be a const, It's weird because the error message hasn't the same accepted formats :thinking:

Reply:
I have made it a const and made sure message containts same formats as here. Right now I added constants to each type individually, I guess same image types are used in many places so it would make sense for one class to constain const's for all the usages, but I am not sure which one. ImageValidator maybe?